GA-worksheet.md

```diff
@@ -7,62 +7,62 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of a header tag is for headers.
 2. What is a Navigation Bar?
+A list of links.
+3. What tag do we use to make a navigation bar? <ul  >
 
-3. What tag do we use to make a navigation bar? <      >
-
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+4.  <ul> Stands for _unordered lists____________________
+    <li> Stands for _list item_____________________
 
 5. What does the <a> tag do?
-
+links another page.
 6. What would the line of HTML below do?
 
 ``` html
 <a href="http://www.google.com">Google</a>
 ```
-
+it would open your browser.
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
-
+lets you keep CSS in a external file and keeps it more organized.
 2. What is normalize.css?
-
+CSS file that makes sure CSS is rendered the same. 
 3. Why do we need to add a padding property to the `<ul>` tag?
-
+because there is padding there, it must be set to 0.
```

Comment:
Lists have default padding on the left to make up for bullet points. We want to put padding on all sides to make it look centered.